knx.js

```diff
@@ -212,6 +212,46 @@ module.exports = function (RED) {
                 case '6':    //8-bit signed value                 1 Byte                  EIS 14        DPT 6    -128...127
                 case '7':    //16-bit unsigned value              2 Byte                  EIS 10        DPT 7    0...65535
                 case '8':    //16-bit signed value                2 Byte                  DPT 8         DPT 8    -32768...32767
+                case '10':   //Time                               3 Byte                  EIS 3         DPT 10    Day [0..7] Hour [0..23] Minutes [0..59] Seconds [0..59]
+                    var day = 0;
+                    var hours = 0;
+                    var minutes = 0;
+                    var seconds = 0;
+
+                    // Get values from object or parse from input value as wire format
+                    if (typeof(value.day) !== 'undefined' &&
+                    typeof(value.hours) !== 'undefined' &&
+                    typeof(value.minutes) !== 'undefined' &&
+                    typeof(value.seconds) !== 'undefined') {
+                        day = value.day;
+                        hours = value.hours;
+                        minutes = value.minutes;
+                        seconds = value.seconds;
+                    } else {
+                        value = parseInt(value);
+                        // Day 3 bit [0..7]
+                        day = (value >> 21) & 0x07
```

Comment:
what if value isNan here after parseInt?
```suggestion
                        if(Number.isNan(day)) {
                            throw new Error('command value is not a proper number');
                        }
                        day = (value >> 21) & 0x07
```

Reply:
There is not `Number.isNan(...)` function. Must be a `Number.isNaN(...)`
Fix typo, please.